### ISSUE TITLE
R version

### ISSUE THREAD

**Issue by ResearchActuary** [casact/cascsim#1]
Some users have commented that the package does not work and/or underperforms under R version 4.0.x. Downgrading the user R installation to R version 3.6.x appears to resolve the issue.

**Comment by joemarker2** [comment]
I believe this issue is resolved.  Kailan Shang and Hai You revised the cascsim file to handle R version 4.0.4.  Below I have included an portion of Kailan's email of March 22, 2021.

I have redone several simulations under R 4.0.4 using the revised cascsim.xlxm spreadsheet.  These were originally done using R 3.5.3 or R 3.6.3.  So far I have not had any problem with R version 4.0.4. that did not already exist with the earlier R versions.

Excerpt from Kailan Shang email of March 22, 2021:

"Hi Joe,

"Hai and I have updated the R codes and EXCEL file to be able to run under R4.0.4, with some of the changes listed below:
1. Removed two libraries (lsei and npsurv) which are no longer needed in R4 packages
2. Changed copula to be independent by default (right not copula simulation with Normal copula is slow under R4.0.4). This can be an area of improvement if the CAS is interested.
3. Changes to VBA codes to deal with different outputs from R4.
4. Rebuilt cascism under R4.0.4.

"The new EXCEL tool " cascsim_R4.xlsm" that has been tested under R4.0.4 can be downloaded @ https://www.reserveprism.com/Simulator2/R4/cascsim_R4.xlsm

"Let us know if that works for you and the RIsk Committee. We have spent 35+ hours on this already and would like to have a formal agreement/contract for further maintenance and enhancement if the CAS is interested. As discussed, many other useful features/models can be added to the existing simulation tool, although the initial scope is defined based on Parodi's paper."
